plugins/modules/web_infrastructure/apache2_module.py

```diff
@@ -218,6 +220,8 @@ def _set_state(module, state):
             module.exit_json(changed=True,
                              result=success_msg,
                              warnings=module.warnings)
+        elif result == 0:
+            module.exit_json(changed=('already' not in stdout), result=success_msg)
```

Comment:
Does it make sense to add spaces around `already`?
```suggestion
            module.exit_json(changed=(' already ' not in stdout), result=success_msg)
```

Reply:
It makes sense. Committed.